.gitignore

```diff
@@ -7,3 +7,4 @@ build
 env
 venv
 .zanata-cache
+container/l10n.conf
```

Comment:
I think we want to add this to git, otherwise it will not be available when the image is built.